### DISCUSSION TITLE
how install pimcore without use composer

### DISCUSSION THREAD

**Discussion by romedenne** [pimcore/pimcore#13088] [Q&A / Support]
hi all,

I encountered some trouble to install pimcore using composer.

My server is in security environment with impossibility to reach external url.
I use suse linux and suse manager to install all specifics modules.

When i run composer install it try to reach packagelist.org but it is refused of course.

suse manager use only repo define for him.

Do you have one solution to be able install pimcore without composer ?
May be do you have rpm format for pimcoe or other solution ?

 thank you by advance for your help.

Michel

**Comment by NiklasBr** [comment]
Check out some *automated deployment* tools like [Jenkins](https://www.jenkins.io/), [Circle CI](https://circleci.com/product/), or [Travis CI](https://www.travis-ci.com/). With those you can build and package a complete installation which is then pushed to your server.

**Reply by romedenne** [reply]
yes thank you for the response but in my case when i want to install anything suse manager is mandatory and nothing else .
Suse manager use only repo define by him and the format is only  rpm package for example.
If pimcore could provide the pimcore installation with rpm format the limitation doesn't exist anymore.

**Reply by dpfaffenbauer** [reply]
@romedenne I think pimcore providing an RPM is out-of-scope. Pimcore works like any other major PHP Framework/Application. It needs composer to be setup, if you need a RPM file to install it on your own server, you have to create that RPM yourself.
